spec/hatchet/getting_started_spec.rb

```diff
@@ -6,4 +6,10 @@
       expect(app.run("ls tmp/cache/assets")).to_not match("sprockets")
     end
   end
+
+  it "works on Heroku-24" do
+    Hatchet::Runner.new("default_ruby", stack: "heroku-24").deploy do |app|
+      expect(app.run("which ruby")).to eq("/app/bin/ruby\n")
```

Comment:
I usually like to trim/strip off trailing/leading whitespace rather than change the assertion. In the event that that command starts emitting more or less whitespace the test won't start to fail. 

```suggestion
      expect(app.run("which ruby").strip).to eq("/app/bin/ruby")
```

Ruby string docs: https://rubyapi.org/2.3/o/string#method-i-strip

You can accept this suggestion directly into the PR, but once you do that you'll have to pull changes from your own branch locally (just FYI, I don't think this needs any additional iteration).